containers/Dockerfile.EESSI-build-node-debian10

```diff
@@ -1,5 +1,6 @@
 ARG cvmfsversion=2.8.2
 ARG awscliversion=1.20.39
+ARG fuseoverlayfsversion=1.8
 
 FROM debian:10.6 AS prepare-deb
```

Comment:
Latest Debian 10.x is at 10.11, worth a bump too? (same below)

Reply:
Done!